tools/download-kafka.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+# Copyright 2018 Ciena Corporation
+#
+# Download and extract a Kafka distribution tarball.
+# To list the supported Kafka versions:
+#
+#     tools/download-kafka.py list
+#
+# To download a given version:
+#
+#     tools/download-kafka.py get <KAFKA_VERSION>
+
+import argparse
+import hashlib
+import os
+from shutil import copyfileobj
+import sys
+import tarfile
+from tempfile import TemporaryFile
+from urllib.request import urlopen
+
+CHUNK_SIZE = 32 * 1024 * 1024
+
+versions = {
+    '0.8.0': (
+        'kafka_2.8.0-0.8.0',
+        'https://archive.apache.org/dist/kafka/0.8.0/kafka_2.8.0-0.8.0.tar.gz',
+        hashlib.md5,  # FIXME
+        bytes.fromhex('59 3E 0C F9 66 E6 B8 CD 1B BF F5 BF F7 13 C4 B3'),
+    ),
+    '0.8.1': (
+        'kafka_2.9.2-0.8.1',
+        'https://archive.apache.org/dist/kafka/0.8.1/kafka_2.9.2-0.8.1.tgz',
+        hashlib.md5,  # FIXME
+        bytes.fromhex('BF 02 96 AE 67 12 4A 76 96 64 67 E5 6D 01 DE 3E'),
+    ),
+    '0.8.1.1': (
+        'kafka_2.9.2-0.8.1.1',
+        'https://archive.apache.org/dist/kafka/0.8.1.1/kafka_2.9.2-0.8.1.1.tgz',
+        hashlib.md5,  # FIXME
+        bytes.fromhex('75 41 ED 16 0F 1B 3A A1 A5 33 4D 4E 78 2B A4 D3'),
```

Comment:
`9897e899ecb3ddbd71705436ed7e568166685b5d168f2c2f852956aacfb80a93b6864c1502c9a4537fc27b587e1fa06e2390c0f4f56d7400fef116245c4f1d81` is the SHA512 for this one...